Example/CustomActionControllers/Twitter.swift

```diff
@@ -97,6 +97,7 @@ open class TwitterActionController: ActionController<TwitterCell, ActionData, Tw
         cellSpec = CellSpec.nibFile(nibName: "TwitterCell", bundle: Bundle(for: TwitterCell.self), height: { _ in 56 })
         headerSpec = .cellClass(height: { _ -> CGFloat in return 45 })
         
+        cancelSpec = .nibFile(nibName: "CancelCell", bundle: Bundle(for: CancelCell.self), height: { _ in 46 })
```

Comment:
I'm totally agree, this should not have been committed. This was added to see that with this actionController could be added a cancel view without crash.